### ISSUE TITLE
Applicants' form-filling UI should meet WCAG 2.1 accessibility standards

### ISSUE THREAD

**Issue by neel-a-mehta** [civiform/civiform#302]
See https://www.w3.org/WAI/standards-guidelines/wcag/ and the City of Seattle's digital style guide: https://app.uxpin.com/design-system/3f410cc070aa87dbff70/city-of-seattle-digital-style-guide

**Comment by baipai** [comment]
This is another good source from the digital gov, [https://accessibility.digital.gov](https://accessibility.digital.gov).
They have some very good examples for the form design and others.

**Comment by jeanniefu** [comment]
Closing this bug since it is very general (though feel free to re-open it if you find it useful to also have this). We had the a11y team perform a full audit against WCAG 2.1. I am currently filing bugs based on the audit (they will all be prefixed with "WCAG 2.1" in the title). Once those bugs are all fixed we should be up to the guidelines!
